.cirrus.yml

```diff
@@ -51,14 +51,6 @@ freebsd_task:
     - pip install "impacket"
   configure_script:
     - autoreconf -fi
-    # Building with the address sanitizer is causing unexplainable test issues due to timeouts
```

Comment:
The commits of this PR need to be cleaned up / squashed and this removal should be its own one.

Reply:
Agreed